vars/fcosKola.groovy

```diff
@@ -27,8 +28,12 @@ def call(params = [:]) {
             {
                 args += "--exttest ${env.WORKSPACE}"
             }
+            def extraArgs = params['extraArgs'];
+            if (extraArgs == null) {
+                extraArgs = ""
+            }
             try {
-                shwrap("cd ${cosaDir} && cosa kola run --parallel 8 ${args}")
+                shwrap("cd ${cosaDir} && cosa kola run --parallel 8 ${args} ${extraArgs}")
```

Comment:
Actually, should we also add a `parallel` argument while we're here? That way the rpm-ostree CI can set it to something it accounted for when calculating pod resource requests.

Reply:
OK done.  I also found the two-argument `.get()` method which cleans up default handling.

That said...I think in the future we should teach kola how to auto-calculate this itself.